src/modules/dashboard/components/Dialogs/CancelStreamingDialog/CancelStreamingDialog.css

```diff
@@ -0,0 +1,198 @@
+@value radioButtonWidth: 340px;
+@value buttonWidth: 160px;
+
+.row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 8px 0 13px;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+  color: var(--temp-grey-blue-7);
+  letter-spacing: 0.5px;
+}
+
+.rowAlt {
+  padding: 11px 0 15px;
+  min-height: 3rem;
+  border-bottom: 1px solid var(--grey-blue-0);
+}
+
+.withoutPadding {
+  padding: 0;
+}
+
+.forceRow {
+  align-items: center;
+  padding-top: 17px;
+  white-space: nowrap;
+}
+
+.forceRow button > div > div {
+  margin-bottom: 4px;
+}
+
+.forceRow form {
+  max-width: 200px;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 9px 0 12px;
+  width: 100%;
+  font-size: var(--size-medium-l);
+  font-weight: var(--weight-bold);
+  line-height: 24px;
+  color: var(--dark);
+  letter-spacing: 0.5px;
+}
+
+.description {
+  font-size: var(--size-normal);
+  line-height: 1.3;
+  color: var(--temp-grey-blue-7);
+  letter-spacing: 0.1px;
+}
+
+.dialogSectionTitle {
+  padding-top: 25px;
+  padding-bottom: 12px;
+  border-bottom: 1px solid var(--grey-blue-0);
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  line-height: 1.4;
+  color: var(--dark);
+  letter-spacing: 0.5px;
+}
+
+.annotations {
+  margin: 24px 0 5px;
+  letter-spacing: 0.5px;
+}
+
+.tokens {
+  padding: 11px 0 15px;
+  border-top: 1px solid var(--grey-blue-0);
+  border-bottom: 1px solid var(--grey-blue-0);
+}
+
+.value {
+  display: flex;
+  align-items: center;
+  font-size: var(--size-smallish);
+  color: color-mod(var(--dark) alpha(85%));
+}
+
+.value span {
+  padding-right: 2px;
+  font-size: var(--size-smallish);
+}
+
+.toggleContainer i {
+  margin: 0 -20px 4px 21px;
+
+  /* next lines are used for the correct tooltip arrow placement */
+  transform: translateX(-25px);
+}
+
+/* this selector is for tooltip wrapper */
+.toggleContainer > div > div:last-child {
+  max-width: 250px;
+  white-space: normal;
+}
+
+/* the following lines center the toggle force row */
+
+.toggleContainer label {
+  display: flex;
+  align-items: center;
+  margin-bottom: 0;
+}
+
+.toggleContainer > div {
+  align-items: center;
+}
+
+.toggleContainer > div > div:first-of-type {
+  height: 16px;
+}
+
+.toggleContainer > div > span {
+  height: 16px;
+}
+
+.toggleContainer > div > span > div {
+  height: 16px;
+}
+
+.userContainer {
+  /* composes: settingsRow; */
```

Comment:
Can this be deleted if it's commented out?

Reply:
i have added the missing styles, which were used in previous dialogs so the composes property can ba used